server/controllers/api.js

```diff
@@ -148,6 +148,10 @@ exports.downloadRepository = function(req, res) {
          */
         function(files, done) {
             return async.map(files, function(file, cb) {
+                //Convert '/' to '\'
```

Comment:
Perhaps the correct solution is to instead use something like `path.join` on the glob above.